webservices/setup.md

```diff
@@ -28,7 +28,7 @@ Make sure everyone else on the team has access
 12. Exit, run bundle
 
 13. https://github.com/IntrepidPursuits/follow-app-server/blob/master/spec/support/helpers/requests.rb
-Copy the contents of this file into spec/support/requests.rb
+Copy the contents of this file into spec/support/helpers/requests.rb [Original link to be updated]
```

Comment:
So I tried to follow the link and I got a 404, I assume that's the reason we need to update it. 

I think this link should work: https://github.com/IntrepidPursuits/club-foreman-server/blob/master/spec/support/helpers/requests.rb which I think all of Intrepid should have access to.